krun.py

```diff
@@ -246,6 +247,12 @@ def main(parser):
         if rc != 0:
             util.fatal("Could not touch config file: " + args.filename)
 
+        if not args.develop and not args.dry_run:
+            info(("Wait %s secs to allow system to cool prior to "
```

Comment:
@ltratt I'm not all that fussed what the default is, we could easily have 60. As for a countdown, I would have thought the log message was enough already.